config/models.yaml

```diff
@@ -0,0 +1,31 @@
+models:
+    cosmo:
+        jobs: ['prepare_data', 'int2lm', 'cosmo', 'post_cosmo']
+        variants: ['spinup']
+        features: ['restart']
```

Comment:
What is the difference between a job, a variant and a feature?

Reply:
`variants` and `features` change the behavior of the chain and are dependent on the used model. I discussed already with @gredvis and I think these two can be merged (i.e., only have `features`).

A job is a python script that is executed when running the chain. The jobs listed here are the default ones, i.e. if no jobs are specified, those are used.